pkg/kubelet/apis/config/helpers_test.go

```diff
@@ -204,6 +204,7 @@ var (
 		"StaticPodURLHeader[*][*]",
 		"MaxOpenFiles",
 		"MaxPods",
+		"MemoryManagerPolicy",
```

Comment:
I'm not familiar with these tests. Should we also have entries for the TopologyManager and CPUManager in here as well? If not, what makes the MemoryManager special that we have it and not the others?

Reply:
It already here:
CPU manager - https://github.com/kubernetes/kubernetes/blob/f170d33e1f320d7a3bcd2ae5a60b7d335ae382f6/pkg/kubelet/apis/config/helpers_test.go#L152
Topology Manager - https://github.com/kubernetes/kubernetes/blob/f170d33e1f320d7a3bcd2ae5a60b7d335ae382f6/pkg/kubelet/apis/config/helpers_test.go#L154

Reply:
Ack